internal/controller/node_reconciler.go

```diff
@@ -100,6 +106,10 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 			continue
 		}
 
+		// Track eviction and node drain events
+		metrics.IncrementEvictionCount(pod.Namespace, pod.Name)
```

Comment:
Could also track this in eviction autoscaler controller when there is a difference between last eviction in spec and status.